src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -705,7 +716,10 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
 
             //PROTOTYPE: should be a wellknown member?
             var ctor = type.UnderlyingTupleType?.InstanceConstructors.FirstOrDefault();
-
+            if ((object)ctor == null)
+            {
+                return BadExpression(node);
```

Comment:
First, an error should be reported, calling BadExpression doesn't do that. Second, the bound arguments should be preserved in the tree (see line 639). Note, if we report an error, we can simply proceed to creating BoundTupleCreationExpression with hasErrors == true and null constructor. The constructor is used only in lowering, which we won't reach in presence of errors.